### PR TITLE
Add !listbio command

### PR DESCRIPTION
**Pull request by GTG3000**
And miscellaneous improvements.

### CODE REVIEW

File: plugins/custom_commands.py [hunk 1]
```diff
@@ -155,10 +157,10 @@ class CustomCommands(BasePlugin):
         else:
             t_count = len([True for i in self.ccs[gid].values() if i["author"] == msg.author.id])
 
-            if msg.author.id not in self.config_manager.config.get("bot_maintainers", []) and\
-                    not msg.author.permissions_in(msg.channel).manage_messages and\
+            if msg.author.id not in self.config_manager.config.get("bot_maintainers", []) and \
+                    not msg.author.permissions_in(msg.channel).manage_messages and \
                     t_count >= self.plugin_config[gid]["cc_limit"]:
```
Comment (medeor413) [inline plugins/custom_commands.py]: Missed this one

Comment (GTG3000) [reply plugins/custom_commands.py]: That was undone by a merge conflict. Ech.

File: plugins/roleplay.py [hunk 2]
```diff
@@ -288,7 +288,11 @@ class Roleplay(BasePlugin):
             t_file = BytesIO()
             await msg.attachments[0].save(t_file)
             try:
-                t_data = json.loads(t_file.getvalue().decode())
+                t_string = t_file.getvalue().decode()
+                t_data = json.loads(t_string)
+            except json.decoder.JSONDecodeError as e:
+                self.logger.exception("Could not decode ccs.json! ", exc_info=True)
```
Comment (medeor413) [inline plugins/roleplay.py]: Misleading exception message

Comment (GTG3000) [reply plugins/roleplay.py]: fixed

**Pull request merged**